keyboards/dztech/dz65rgb/keymaps/chocol8/keymap.c

```diff
@@ -1,55 +1,25 @@
 #include QMK_KEYBOARD_H
 
 enum tofu68_layers {
-	_MAQWERTY,
-    _LIQWERTY,
+    _QWERTY,
 	_FNM
 };
 
-enum tofu68_keycodes {
-	MAQWER = SAFE_RANGE,
-    LIQWER
-
-};
-
 #define FNM MO(_FNM)
 
 const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
-	[_MAQWERTY] = LAYOUT_65_ansi(
+	[_QWERTY] = LAYOUT_65_ansi(
 		KC_GESC, KC_1,     KC_2,     KC_3,   KC_4,    KC_5,    KC_6,    KC_7,    KC_8,    KC_9,    KC_0,     KC_MINS,  KC_EQL,  KC_BSPC, KC_GRV,  \
 		KC_TAB,  KC_Q,     KC_W,     KC_E,   KC_R,    KC_T,    KC_Y,    KC_U,    KC_I,    KC_O,    KC_P,     KC_LBRC,  KC_RBRC, KC_BSLS, KC_DEL, \
 		KC_GRV,  KC_A,     KC_S,     KC_D,   KC_F,    KC_G,    KC_H,    KC_J,    KC_K,    KC_L,    KC_SCLN,  KC_QUOT,           KC_ENT,  KC_PGUP, \
-		KC_LSPO, KC_Z,     KC_X,     KC_C,   KC_V,    KC_B,    KC_N,    KC_M,    KC_COMM, KC_DOT,  KC_SLSH,  KC_RSPC,           KC_UP,   KC_PGDN,  \
+		KC_LSPO, KC_Z,     KC_X,     KC_C,   KC_V,    KC_B,    KC_N,    KC_M,    KC_COMM, KC_DOT,  KC_SLSH,  KC_RSFT,           KC_UP,   KC_PGDN,  \
```

Comment:
`KC_LSPO` paired with a regular `KC_RSFT`? Was this change deliberate?

Reply:
not deliberate. fixed now. thanks @noroadsleft